grapher/graph/schema.resolvers.go

```diff
@@ -80,8 +93,42 @@ func (r *queryResolver) PopularDecks(ctx context.Context, first *int, after *str
 	return out, nil
 }
 
-func paginationFromInput(first *int, after *string, last *int, before *string) *v1.Pagination {
-	var out v1.Pagination
+func (r *queryResolver) Cards(ctx context.Context, input model.CardsInput) ([]*model.Card, error) {
+	var maxCards int
+	if input.MaxCards > 0 {
+		maxCards = input.MaxCards
+	}
+
+	res, err := r.DealerClient.Deal(ctx, &v1Dealer.DealRequest{
+		UserId:        r.getUserID(ctx),
+		DeckId:        input.DeckID,
+		NumberOfCards: uint32(maxCards),
+	})
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+
+	cards := make([]*model.Card, 0, len(res.CardIds))
+
+	for _, cardID := range res.CardIds {
+		c, err := r.CardLoader.Load(ctx, cardID)
+		if err != nil {
+			if errors.Is(err, ErrNoResult) {
+				continue
```

Comment:
`ErrNoResult` happens when the batching function of the loader did not return a result for the given ID.
In theory IDs from Dealer should always be valid and this case should never happen. Ideally  apart from only `continue` we would log this instead.